waspc/cli/src/Wasp/Cli/Interactive.hs

```diff
@@ -83,18 +98,30 @@ askToChoose question options = do
     showIndexedOptions = intercalate "\n" $ showIndexedOption <$> zip [1 ..] (NE.toList options)
       where
         showIndexedOption (idx, option) =
-          Term.applyStyles [Term.Yellow] indexPrefix
-            <> Term.applyStyles [Term.Bold] (showOption option)
-            <> (if isDefaultOption option then " (default)" else "")
-            <> showDescription option (length indexPrefix)
+          concat
+            [ indexPrefix,
+              optionName,
+              tags,
+              optionDescription
+            ]
           where
-            indexPrefix = showIndex idx <> " "
+            indexPrefix =
+              Term.applyStyles
+                ([Term.Yellow] <> whenDefault [Term.Bold, Term.Underline])
+                (showIndex idx)
+                <> " "
+            optionName = Term.applyStyles [Term.Bold] (showOption option)
+            tags = whenDefault (Term.applyStyles [Term.Yellow] " (default)")
+            optionDescription = showDescription option
+            whenDefault xs = if isDefaultOption option then xs else mempty
```

Comment:
I enriched the styling of options a bit, so now `(default)` tag is also yellow, and the index `[i]` of the default option is bold and underlined.

Reply:
![Screenshot 2023-12-19 at 12 40 28](https://github.com/wasp-lang/wasp/assets/2223680/f08f1d67-b7f7-40e6-9fb7-c60849a2f8d8)

I'd remove the Bold and Underline for the index, it confused me for a second that I can move the options with my keyboard which I couldn't 😢  Until we have that, let's maybe not do anything different for the index. 

I like the yellow `(default)` 👍

Reply:
Good point, done!